internal/playlist_creator/FileData_test.go

```diff
@@ -0,0 +1,255 @@
+package playlist_creator
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"playlistCreator/internal/playlist_creator/config"
+	"testing"
+)
+
+func TestFileData_addFileExtension(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name                   string
+		extensions             []string
+		expectedExtensionsSize int
+		expectedExtensions     map[string]int
+	}{
+		{
+			name:                   "Should add an extension that does not exist",
+			extensions:             []string{"abc"},
+			expectedExtensionsSize: 1,
+			expectedExtensions:     map[string]int{"abc": 1},
+		},
+		{
+			name:                   "Should add multiple extensions",
+			extensions:             []string{"abc", "123"},
+			expectedExtensionsSize: 2,
+			expectedExtensions:     map[string]int{"abc": 1, "123": 1},
+		},
+		{
+			name:                   "Should increment extensions",
+			extensions:             []string{"abc", "123", "abc"},
+			expectedExtensionsSize: 2,
+			expectedExtensions:     map[string]int{"abc": 2, "123": 1},
+		},
+	}
+
+	for _, testData := range tests {
+		testData := testData
+		t.Run(testData.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I don't think you need this here. this should be Parallel already.
`t.Run` makes the test run in a diff goroutine.

Reply:
https://www.codingexplorations.com/blog/parallel-testing-in-go